blocks/core/ff_container/ff_container-expandable/ff_container-expandable-component/_ff_container-expandable-component-renderer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+var React = require('react');
+
+var Expandable = require('./ff_container-expandable-component');
+
+var data = {
+	expdanded_text: 'Hide content',
```

Comment:
Hi Simone, purely since we're trying to be consistent across projects, can you use camelCasing for js vars? Thanks!